pantheon/src/main/java/tech/pegasys/pantheon/chainimport/ChainData.java

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Is there a way to parse the blocks directly into a sequence without the need of a separate class?

Reply:
Maybe?  But I think the structure is pretty clear as is and it provides a place to add additional data if we need it in the future.